### ISSUE TITLE
Help with CSS

### ISSUE THREAD

**Issue by gabbyhershey** [gabbyhershey/SnailPal#4]
Can some one add some custom CSS please

**Comment by ramonborges15** [comment]
@gabbyhershey can you explain more about it?

**Comment by baycak** [comment]
Can you elaborate more about this issue?

**Comment by gabbyhershey** [comment]
yes, please checkout the code and write some css as you think would fit, and make a PR again the UX branch with some screenshot
